source/index.html.md

```diff
@@ -225,16 +242,16 @@ curl \
 ```
 `POST /users`
 
-This method is create a new user in the Betable API and should be called at the same time that a new user is created in the vendor's platform.
+This method is create a new user in the Betable API and should be called at the same time that a new user is created in the partner's platform.
 
-### Query Parameters
+### Request Parameters
 
 | Parameter | Type | Example | Description |
 | ---| ---| ---| --- |
 | first\_name | string | John | The user's first name. |
 | last\_name | string | Doe | The user's last name. |
 | email | string | john@doe.com | The user's email. |
-| external\_user\_id | string | vendor-user-id-1 | A unique ID that can be used to identify the user's account in the PGP. |
+| external\_user\_id | string | partner-user-id-1 | A unique ID that can be used to identify the user's account in the partner platform. |
```

Comment:
I think this is one of the remaining PGPs, is this term of "partner platform" ok?